docs/file-scrapers.md

```diff
@@ -36,6 +36,13 @@ Go to https://www.erlang.org/downloads and download the HTML documentation file.
 
 ### GCC
 ### GNU Fortran
+Go to https://gcc.gnu.org/onlinedocs/ and download the HTML tarball of Fortran manual or run the following commands to download the tarball:
+
+```sh
+mkdir docs/gnu_fortran~$VERSION; \
+curl https://gcc.gnu.org/onlinedocs/gcc-$RELEASE/gfortran-html.tar.gz | \
+tar --extract --gzip --strip-components=1 --directory=docs/gnu_fortran~$VERSION
+```
```

Comment:
I'm trying to standardise the variables `$VERSION`, `$RELEASE`, as well as combined `curl`/`tar --extract` to download/extract in one go. Efficiency is king! :-)